README.md

```diff
@@ -273,6 +273,18 @@ And you can change the JSON key that the serializer should use for a particular
 The `url` declaration describes which named routes to use while generating URLs
 for your JSON. Not every adapter will require URLs.
 
+## Pagination
+
+If you want pagination links in your response, specify it in the `render`
+
+```ruby
+  render json: @posts, pagination: true
```

Comment:
Thanks for this.  Did you try subclassing the `ArraySerializer`? e.g. https://github.com/rails-api/active_model_serializers#2-for-an-array-resource and just setting the meta?  Have you seen pagination gems such as https://github.com/davidcelis/api-pagination ?